shell/client/shell.html

```diff
@@ -1151,6 +1151,91 @@ <h3 id="app-stats">Apps</h3>
   </div>
 </template>
 
+<template name="referrals">
+  {{#if quotaEnabled}}
+  {{setDocumentTitle}}
+  {{>title "About Sandstorm"}}
+  <div class="about">
+    {{#if isPaid}}
+    <h1>Get up to 30GB of free space when your friends share on Oasis!</h1>
+    {{else}}
+    <h1>Get unlimited grains &amp; up to 2GB of free space when
+      your friends share on Oasis!</h1>
+    {{/if}}
+
+    <h2>Your reward</h2>
+
+    {{#if isPaid}}
+    <p>
+      Once someone clicks a sharing link that your friend created,
+      we'll reward you with 2 GB of free storage. You can earn <strong>
+      up to 30GB of free storage</strong> this way!
+    </p>
+    {{else}}
+    <p>
+      Once someone clicks a sharing link that your friend created,
+      we'll reward you with <strong>unlimited grains</strong> and 50
+      MB of free storage.
+      Tell them to share an Etherpad grain or a
+      Game of Y game with you.
+    </p>
+
+    <p>You'll also earn 50 MB per person you successfully
+      refer after the first, <strong>up to 2 GB of free storage.</strong></p>
+    {{/if}}
+
+    <h2>Sharing is referring</h2>
+
+    <p>The best way to refer a friend is to <a href="/grains">open any
+        grain</a> and click <strong>Share access</strong> in the
+        Sandstorm menu. When you send your friend a sharing link, and
+        they create an account, you've referred them. We'll reward you
+        when they share a first grain for the first time.</p>
+
+    <p>Get started by <a href="/grains">viewing your list of
+        grains</a>.</p>
+
+    <!--
+
+    TODO(soon): Integrate with Powerbox identity picker to get referral link
+                sending to exist and to look nice.
+
+    <h2>Or send a referral link</h2>
+
+    <p>You can type their email address here:</p>
+
+    <form
+    <textarea style="width: 100%" placeholder="Add emails"></textarea>
+
+    -->
+
+    <h2>The fine print</h2>
+
+    <!--
+
+    TODO(soon): Create a referral link mechanism.
+
+    <p>If you prefer, you can share this referral link:
+      <a href="#">https://oasis.sandstorm.io/r/miequ3ie</a></p>
+
+    -->
+
+    <p>Your referral bonus lasts until <strong>1 year after the
+    referral.</strong> This referral program will go away after a
```

Comment:
"This referral program will go away after a limited time." sounds like I could lose my bonus storage at any arbitrary time, contrary to the previous sentence.

I'm not sure if it's necessary to specify that the program itself may go away (assuming bonuses received through it still remain for 1 year). But I suppose another way to say it could be: "The terms of this referral program may change without warning; however, such changes will not affect bonuses you have already received."